### PR TITLE
AutoEdits: General cleanup of file

### PR DESCRIPTION
**Pull request by DannyS712**
Alphabetize tools lists, separate global overrides from local tools, move commons out of the wikipedias and group it with wikidata, etc. No change in functionality should have occurred.
Bug: T222327

### CODE REVIEW

**Comment by DannyS712** [comment]
@MusikAnimal I can't tell why this fails - every site I asked says that the YAML is valid...

File: config/semi_automated.yml [hunk 1]
```diff
@@ -161,314 +169,265 @@ parameters:
...
-            regex: '\[\[MediaWiki:Gadget-SettingsManager.js\|SettingsManager'
-            link: MediaWiki:Gadget-SettingsManager.js
-        RotateLink:
-            regex: '\(\[\[Help:RotateLink\|Script'
-            link: Help:RotateLink
-    de.wikipedia.org:
+        WPCleaner:
+            link: ويكيبيديا:وب_كلينر
+            label: وب_كلينر
+        de.wikipedia.org:
```
Comment (MusikAnimal) [inline config/semi_automated.yml]: `de.wikipedia.org:` is indented one tab too many. This is why the test was failing :)

**Review by MusikAnimal — CHANGES_REQUESTED** [review]
> MusikAnimal I can't tell why this fails - every site I asked says that the YAML is valid...-

See the comment. I guess the other YAML validators weren't checking for duplicate keys! Because that would not be valid.

File: config/semi_automated.yml (hunk 1 above)
Comment (DannyS712) [reply config/semi_automated.yml]: Done, but that doesn't appear to have been the only issue...

Comment (MusikAnimal) [reply config/semi_automated.yml]: This time it was a fluke with Scrutinizer. I triggered another build and it passed.

I'll try to get to doing a full review soon.

**Review by MusikAnimal — APPROVED** [review]
This was very tedious to review, but I manually went through everything and it looks good. Thank you for making separate commits, that helped (though I will still squash them when merging).

**Pull request merged**